new-log-viewer/src/components/StatusBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import Button from "@mui/joy/Button";
+import Sheet from "@mui/joy/Sheet";
+import Typography from "@mui/joy/Typography";
+
+
+interface StatusBarProps {
+    handleCopyLinkButtonClick: () => void;
+    logEventNum: number | null;
+    numEvents: number | null;
+}
+
+export const StatusBar: React.FC<StatusBarProps> = ({handleCopyLinkButtonClick, logEventNum, numEvents}) => {
```

Comment:
```suggestion
const StatusBar = ({handleCopyLinkButtonClick, logEventNum, numEvents}: StatusBarProps) => {
```

Then we don't need the 
```
import React from "react";
```